limit-break/set.cpp

```diff
@@ -38,5 +38,8 @@ void set_init()
 	WriteData((int*)0x0046BF3D, SET_COUNT);
 	WriteData((int*)0x0046BF44, SET_COUNT);
 
+	WriteData((short**)0x0046C1DB, &set_table[SET_COUNT].Flags);
+	WriteData((int*)0x0046C1E0, (SET_COUNT + 1));
```

Comment:
These two are reversed - `set_table[SET_COUNT]` should be `set_table[SET_COUNT - 1]`, and `SET_COUNT + 1` should simply be `SET_COUNT`